traitsui/wx/table_editor.py

```diff
@@ -18,65 +18,44 @@
 """ Defines the table editor for the wxPython user interface toolkit.
 """
 
-#-------------------------------------------------------------------------
-#  Imports:
-#-------------------------------------------------------------------------
-
 from __future__ import absolute_import
 
 from operator import itemgetter
 
 import wx
 
-from traits.api \
-    import Int, List, Instance, Str, Any, Button, Tuple, \
-    HasPrivateTraits, Bool, Event, Property
-
-from traitsui.api \
-    import View, Item, UI, InstanceEditor, EnumEditor, Handler, SetEditor, \
+from pyface.ui.wx.grid.api import Grid
```

Comment:
nit : it looks like the rest of the imports are ordered alphabetically but this one is out of order! (_i'm out of order? no you're out of order!_)